job_service/adapter/local_storage.py

```diff
@@ -9,44 +10,57 @@
 ARCHIVE_DIR = INPUT_DIR / "archive"
 
 
+def _has_data(tar, dataset_name):
+    chunks_count = len(
+        [
+            name
+            for name in tar.getnames()
+            if name.startswith("chunks/") and name.endswith(".csv.encr")
+        ]
+    )
+    data_files = [
+        f"{dataset_name}.symkey.encr",
+        f"{dataset_name}.md5",
+        "chunks",
+    ] + [f"chunks/{i}.csv.encr" for i in range(1, chunks_count + 1)]
```

Comment:
I think it is enough to check if `chunks_count > 0`.

Reply:
Could it be enough to check:
```py
return "chunks" in tar.getnames()
```
It might seem minimal, but the complete check for correctness (md5 etc) is done in job_executor regardless, so I think it is enough to say "this seems to have data" at this stage. (just like we say "it seems like it has metadata" even though it could technically be an empty file (later validated by job-exec of course))